solutions/cses-1091.mdx

```diff
@@ -51,3 +56,69 @@ int main(){
 	return 0;
 }
 ```
+
+</CPPSection>
+
+<JavaSection>
+<Warning>
+Due to the way CSES's grader works (allowing 1 second for all problems + being slow), test cases 5 and 8 TLE (time limit exceeded). USACO gives Java test cases 4 seconds, so this isn't representative of what competing with Java will be like. 
```

Comment:
Try reading the tickets and max prices through io.nextLine(), and then splitting them into integer arrays. I've generally found this to be fast enough to pass CSES.

Reply:
From what I can see, there isn't a nextLine function in Kattio, there's only a next function that returns the next token.  I have made some improvements to reduce runtime, they haven't fixed the TLE issue.

Reply:
Never mind, I just used a faster reader (hidden through codesnip). Now all tests pass.